packages/sdk-core/src/Framework.ts

```diff
@@ -250,10 +251,47 @@ export default class Framework {
 
     /**
      * @dev Load a `SuperToken` class from the `Framework`.
-     * @param address the `SuperToken` address
+     * @param tokenAddressOrSymbol the `SuperToken` address or symbol
```

Comment:
Should mention that the symbol has to be listed for this to work?

Reply:
true